buildReleaseNotes.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+import json
+
+subprocess.call(['sh', './getActionRuns.sh'])
```

Comment:
since all you're doing is curling, maybe just use the requests library?  that way the entire process is inside a single python file and can work even for future windows-OS'd devs stuck working on our build pipeline locally.  that way you won't even need the actionRuns.json file